src/ray/core_worker/core_worker.cc

```diff
@@ -1813,12 +1817,11 @@ Status CoreWorker::CreateActor(const RayFunction &function,
         "Async actor is currently not supported for the local mode");
   }
 
-  RAY_CHECK(job_config_ != nullptr);
   bool is_detached = false;
   if (!actor_creation_options.is_detached.has_value()) {
     /// Since this actor doesn't have a specified lifetime on creation, let's use
     /// the default value of the job.
-    is_detached = job_config_->default_actor_lifetime() ==
+    is_detached = worker_context_.GetCurrentJobConfig().default_actor_lifetime() ==
```

Comment:
Could we send JobConfig only for the actor constructor task, and not further actor method calls?

Reply:
done